src/pages/Main.js

```diff
@@ -82,9 +82,8 @@ class Main extends Component {
     }
```

Comment:
higher order component, react single page apps. One page handles all logic, instead of sending you to different pages